R/code.R

```diff
@@ -1,7 +1,8 @@
 # The rest of this documentation is in case.R
 #' @export
 #' @rdname new_item
-vt_use_test_code <- function(name, username = vt_username(), open = interactive()) {
+vt_use_test_code <- function(name, username = vt_username(), open = interactive(), pkg = ".",
```

Comment:
In the develop version of the package, `pkg` for all the vt_use_* functions is no longer necessary. you can remove that argument from these functions and from `vt_add_file_to_config()`.